README.md

```diff
@@ -1,7 +1,434 @@
 Jenkins LDAP Plugin
 ===================
 
-Read more: [https://wiki.jenkins-ci.org/display/JENKINS/LDAP+Plugin](https://wiki.jenkins-ci.org/display/JENKINS/LDAP+Plugin)
+*Note*: This plugin was part of the Jenkins core until 1.468. After
+that, it was split out into a separately-updateable plugin. However, for
+backwards compatibility purposes, subsequent core releases still bundle
+it. If you do not use this plugin at all, you can simply disable it.
+
+# Description
```

Comment:
There should be only one top level heading on a page, in this case the heading can be removed